content/blog/2025-05-27-welcoming-almalinux-10.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "AlmaLinux OS 10 - usability without compromising compatibility"
+type: blog
+author:
+ name: "Eduard Abdullin"
+ bio: "Release and Automation Engineer"
+ image: /users/eduard-abdullin.jpg
+date: '2025-05-22'
+images:
+  - /blog-images/2025/almalinux10.png
+post:
+    title: "Read more about how AlmaLinux OS 10 is the perfect marriage of maintaining compatibility and serving our community!"
+    image: /blog-images/2025/almalinux10.png
+
+---
+
+# AlmaLinux OS 10.0 Stable Now Available 
+
+The AlmaLinux OS Foundation is proud to announce the general availability of [AlmaLinux OS 10.0](https://mirrors.almalinux.org/isos.html) codenamed "Purple Lion"!
+
+# AlmaLinux OS Improvements - usability without compromising compatibility
+
+The goal of AlmaLinux OS is to support our community, and AlmaLinux OS 10 is the best example of that yet. With an unwavering eye on maintaining compatibility with Red Hat Enterprise Linux (RHEL), we have made small improvements to AlmaLinux OS 10 that target specific sections of our userbase. They are for users who require them and know how to use them. Anyone using AlmaLinux OS and anticipating RHEL compatibility is in safe hands, and won't 
+
+Like we discussed before, AlmaLinux OS 10.0 has been built from our brand new upstream, AlmaLinux OS Kitten. If you have been watching the discussions that ALESCo has been having, you'll know that Kitten brought with it a ton of new things that our community needs. 
+
+## Supporting developers by enabling frame pointers
+For software developers, frame pointers are critical to diagnosing and optimizing their applications. For those developers that use AlmaLinux as their base, enabling frame points is a pain point - one that we are happy to help ease. 
```

Comment:
The pain point would be not having frame pointers.

Reply:
Thanks, sir. I spent a bunch of time trying to digest a bunch of guides to figure out how to phrase this. :D